src/utils/helpers.ts

```diff
@@ -167,3 +167,47 @@ export const parseDate = (dateString: string): Date => {
   const [yyyy, mm, dd] = dateString.split('-').map((str) => parseInt(str));
   return new Date(yyyy, mm - 1, dd);
 };
+
+/**
+ * Returns the value after checking if its value is not undefined.
+ *
+ * @remarks
+ * This method is part of the {@link core-library#Statistics | Statistics subsystem}.
+ *
+ * @param key - The key is of type string
+ * @param value - The value can be of any type like string, boolean, object, number.
+ *
+ */
+export const StringifyFormReplacer = (key: string, value: any): unknown => {
```

Comment:
Could we add some type of unit test for this? Maybe something that compares the returned value from `StringifyFormReplace` with an expected value, just to make sure the objects are equivalent?

Reply:
yeah sure